xapi-xenopsd-xc.opam

```diff
@@ -45,6 +45,7 @@ depends: [
   "xenctrl"
   "xenstore"
   "xenstore_transport"
+  "xentoollog"
```

Comment:
Looks like another rebasing mistake.  This was specifically deleted recently due to it having been deleted upstream.